src/lang/proc_macros/db.rs

```diff
@@ -97,14 +97,13 @@ pub fn get_derive_expansion(
     params: ExpandDeriveParams,
 ) -> ProcMacroResult {
     db.get_stored_derive_expansion(params.clone().into()).unwrap_or_else(|| {
-        let token_stream = params.item.clone();
-
         if let Some(client) = db.proc_macro_server_status().ready() {
             client.request_derives(params);
         }
 
         ProcMacroResult {
-            token_stream: token_stream_v2_to_v1(&token_stream),
+            // We don't remove the original item for derive macros, so return nothing.
```

Comment:
If we don't remove anything, there should be the original item here, why do you return an empty string instead?